Content.Server/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -317,5 +323,46 @@ private void PerformReaction(ReactionPrototype reaction, int unitReactions)
             //Play reaction sound client-side
             _audioSystem.Play("/Audio/effects/chemistry/bubbles.ogg", Owner.Transform.GridPosition);
         }
+
+        /// <summary>
+        /// Called when the owner of this component is attacked by
+        /// another entity. Currently used for click solution transfer behavior.
+        /// Eg: Clicking one beaker with another to transfer chems from one
+        /// to the other.
+        /// </summary>
+        /// <param name="eventArgs">Attack event args</param>
+        /// <returns></returns>
+        bool IAttackBy.AttackBy(AttackByEventArgs eventArgs)
```

Comment:
I feel like this behavior may be better suited for a separate component. At least in SS13 it's a property of `/obj/item/reagent_containers/glass`

Reply:
I can stick it in a mostly empty component that just has `IAttackBy` and yaml config for transfer amount. Only downside of this is that there's slightly more work required to make a container that can do transfers. Now you'll need to have a `SolutionComponent` with the right capability flags for pour in/out and this new component. Not a big deal I think, can just add some comments to the beaker yml so anyone making new ones know.

Probably good so we don't keep adding code onto SolutionComponent and make it into a monster.